### PR TITLE
Feature/post list

### PR DESCRIPTION
**Pull request by curious-curie**
## What does this PR do?

PostList 구현 및 api 연결 위한 background 작업~~~~

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code Style Update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: frontend/src/apis/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+// import history from '@/history';
+
+const instance = axios.create({
+  baseURL: 'http://127.0.0.1:8000/api/'
+});
```
Comment (GooJinSun) [inline frontend/src/apis/index.js:6]: @curious-curie 요건 어떻게 쓰면 되나요?
[axios interceptor](https://velog.io/@skyepodium/axios-%EC%9D%B8%ED%84%B0%EC%85%89%ED%84%B0%EB%A1%9C-API-%EA%B4%80%EB%A6%AC%ED%95%98%EA%B8%B0) 요걸 의도한건가요??

File: frontend/src/modules/post.js [hunk 2]
```diff
@@ -1,12 +1,65 @@
+// import axios from '../apis';
+import { mockFriendFeed } from '../constants';
+
+export const GET_FRIEND_POSTS_REQUEST = 'post/GET_FRIEND_POSTS_REQUEST';
+export const GET_FRIEND_POSTS_SUCCESS = 'post/GET_FRIEND_POSTS_SUCCESS';
+export const GET_FRIEND_POSTS_FAILURE = 'post/GET_FRIEND_POSTS_FAILURE';
```
Comment (GooJinSun) [inline frontend/src/modules/post.js:6]: 이 패턴 너무 좋은 것 같아요!!👍🏻

**Review by GooJinSun — APPROVED** [review]

File: frontend/src/modules/post.js (hunk 2 above)
Comment (curious-curie) [reply frontend/src/modules/post.js:6]: 구욷

File: frontend/src/apis/index.js (hunk 1 above)
Comment (curious-curie) [reply frontend/src/apis/index.js:6]: interceptor 나중에 쓰면 편하지 않을까????? 401 (unauthorized 비슷한) 에러 같은거 한꺼번에 처리하게!!!!!

Comment (curious-curie) [reply frontend/src/apis/index.js:6]: 그리고 axios.get같은거 할 때 full url 다 안쓰구 /user/login 이런식으로만 쓸 수 있는게 편해서!!! 다른 방법 있음 알려줘!

**Pull request merged**